### DISCUSSION TITLE
Hide Title of a Library Panel

### DISCUSSION THREAD

**Discussion by bob454522** [grafana/grafana#37906] [Dashboards, Panels & Visualizations]
Library Panels are an amazing new feature added in v8 grafana!

However, in contrast to "non-Library Panels" - If a panel has no title (and is a library panel) i dont see a way to hide its "title bar": (ie this part in red):
![image](https://user-images.githubusercontent.com/60830628/129422176-c49da0d9-1d90-4342-82a8-8a9b8b82429b.png)

If i set a library panel to NO / Blank title, it will revert back to its prior title (ie as if a blank title is not supported)-  
**Perhaps the way to fix this is to offer a switch, in the panel options, to hide the title.**

(i do understand why each Library Panel must have a title of some kind,  so as to differentiate them).

thanks,
LOVE GRAFANA!

**Comment by szerwi** [comment]
I'd also like the option to disable library panel's icon.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
